client/components/settings-group-card/index.js

```diff
@@ -0,0 +1,28 @@
+/**
+ * External dependencies
+ */
+import React, { PropTypes } from 'react';
+import classnames from 'classnames';
+
+/**
+ * Internal dependencies
+ */
+import CompactCard from 'components/card/compact';
+import FormSectionHeading from 'components/forms/form-section-heading';
+
+const SettingsGroupCard = ( { heading, children } ) => {
```

Comment:
I'm not sure a separate component is worth it here. It's used only 5 times, and makes it harder to understand what's going on in the components that use it.